scripts/clean-sc.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+SCRIPTS_PATH="$(dirname "$(readlink -f "$0")")"
+
```

Comment:
Maybe add a prompt asking the user that they are sure that they want to do this :)?

Reply:
Right, yes I though about that earlier as well. Will add that.

Reply:
Done